libs/wire-api/src/Wire/API/Routes/Public/Galley.hs

```diff
@@ -689,17 +814,37 @@ type MessagingAPI =
         :> ZConn
         :> "conversations"
         :> Capture "cnv" ConvId
-        :> QueryParam "ignore_missing" IgnoreMissing
-        :> QueryParam "report_missing" ReportMissing
         :> "otr"
         :> "messages"
-        :> ReqBody '[Servant.JSON, Proto] NewOtrMessage
+        :> QueryParam "ignore_missing" IgnoreMissing
+        :> QueryParam "report_missing" ReportMissing
+        :> ReqBody '[JSON, Proto] NewOtrMessage
         :> MultiVerb
              'POST
              '[Servant.JSON]
              (PostOtrResponses ClientMismatch)
-             (Either (MessageNotSent ClientMismatch) ClientMismatch)
+             (PostOtrResponse ClientMismatch)
     )
+    :<|> Named
+           "post-otr-broadcast-unqualified"
+           ( Summary "Broadcast an encrypted message to all team members and all contacts (accepts JSON or Protobuf)"
+               :> Description PostOtrDescriptionUnqualified
+               :> ZLocalUser
+               :> ZConn
+               :> CanThrow TeamNotFound
+               :> CanThrow BroadcastLimitExceeded
+               :> "broadcast"
+               :> "otr"
+               :> "messages"
+               :> QueryParam "ignore_missing" IgnoreMissing
```

Comment:
Could it be that the Swagger docs of `ignore_missing` and `report_missing` are lost?

Reply:
They are already contained in the description of the endpoint on line #831: https://github.com/wireapp/wire-server/pull/2016/files#diff-006eafb358b9b080c8fb043d4a24e4575936227b05ae3c8de010af77e1be0febR831.

Reply:
Ah, I see :+1: